config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -1,4 +1,34 @@
 presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-gci-gce-ipvs
+    always_run: false
+    optional: true
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - args:
+        - --timeout=170
+        - --bare
+        - --scenario=kubernetes_e2e
+        - --
+        - --check-leaked-resources
+        - --env=KUBE_PROXY_MODE=ipvs
+        - --extract=ci/latest
+        - --gcp-master-image=gci
+        - --gcp-node-image=ubuntu
+        - --gcp-zone=us-west1-b
+        - --ginkgo-parallel=30
+        - --provider=gce
+        - --test_args=--ginkgo.skip=\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
+        - --timeout=150m
+        image: gcr.io/k8s-testimages/kubekins-e2e:v20200324-0897dba-master
+    annotations:
+      testgrid-num-failures-to-alert: '6'
```

Comment:
This was mostly copied from the current periodic job, guessing we don't need the testgrid alerts for presubmits?

Reply:
yep

Reply:
okay, removed